addons/event_sale/tests/test_event_sale.py

```diff
@@ -494,6 +494,34 @@ def test_ticket_price_with_pricelist_and_tax(self):
         })
         self.assertEqual(so.amount_total, 660.0, "Ticket is $1000 but the event product is on a pricelist 10 -> 6. So, $600 + a 10% tax.")
 
+    def test_event_sale_price_subtotal(self):
```

Comment:
I don't think a full-blown test is necessary here.

You're basically testing that a domain passed to a read_group is applied, this is a base ORM feature, it doesn't really require anything special in terms of code / logic in our own module.